docs/_includes/usage/cf-layout/usage.md

```diff
@@ -1290,208 +1290,181 @@ height of the next parent using `position: relative;`. This means that the
 
 
 ## Featured content module
-{% comment %}
-TODO: Revist the naming of these three modules. The layouts are more about
-positioning the image and less about the type of content they host.
-{% endcomment %}
 
-Featured content module, like a hero, consists of headline/text/optional call
-to action along with a visual. It is intended to be used in a main content
-column next to a sidebar.
+Featured content modules, like a hero, consist of headline and description,
+an optional call to action, and a static or video visual.
+They are intended to be used in a main content column next to a sidebar.
 
 Text is full width & displayed above the visual in the default/mobile view.
```

Comment:
Minor nitpick, but why are we using ampersands here? Probably would be stylistically better to spell out `and`.

Reply:
:+1: Updated in 43458fae.